pyproject.toml

```diff
@@ -93,28 +89,73 @@ write_to = "src/nina/version.py"
 # Tools Configuration #
 #######################
 
+[tool.black]
+line-length = 80
+target-version = ['py39']
+include = '\.pyi?$'
+exclude = '''
```

Comment:
Maybe this could be rewritten to include only what we want to format, instead of exclude all cruft. 🤔